pkg/occlient/occlient.go

```diff
@@ -17,6 +17,28 @@ import (
 
 const ocRequestTimeout = 1 * time.Second
 
+// ocpath stores the path to oc binary
+var ocpath string
+
+var isInitialized bool
+
+func initialize() {
+	var err error
+	ocpath, err = getOcBinary()
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
```

Comment:
it should return error rather than just exit whole program. 
The same for all the other `os.Exit(1)` in this function

Reply:
yeah, borrowed it from the previous `init()` implementation since that does not return an error, but yeah, makes sense, fixed now